projects/plugins/jetpack/modules/contact-form/admin.php

```diff
@@ -1184,74 +1292,112 @@ public static function export_to_gdrive() {
 		$has_valid_connection = Jetpack_Google_Drive_Helper::has_valid_connection( $user_id );
-		// hint user to connect at 'https://wordpress.com/marketing/connections'
-
-		$nonce_name = 'feedback_export_nonce';
 
-		$attributes = array(
-			'data-nonce-name' => $nonce_name,
-		);
-
-		if ( ! $has_valid_connection ) {
-			$attributes['disabled'] = 'disabled';
+		if ( $has_valid_connection ) {
+			$button_html = get_submit_button(
+				esc_html( __( 'Export', 'jetpack' ) ),
+				'primary export-button export-gdrive',
+				'jetpack-export-feedback-gdrive',
+				false,
+				array( 'data-nonce-name' => $this->export_nonce_field )
+			);
+		} else {
+			$button_html = sprintf(
+				'<a href="https://wordpress.com/marketing/connections" class="button button-primary export-button export-gdrive" title="%s" rel="noopener noreferer" target="_blank">%s</a>',
+				esc_attr( __( 'connect to Google Drive', 'jetpack' ) ),
+				esc_html( __( 'Connect Google Drive', 'jetpack' ) )
+			);
 		}
 
-		$button_html = get_submit_button(
-			__( 'Export to Google Drive', 'jetpack' ),
-			'primary',
-			'jetpack-export-feedback-to-gdrive',
-			false,
-			$attributes
-		);
-
-		$button_html .= wp_nonce_field( 'feedback_export', $nonce_name, false, false );
 		?>
-		<script type="text/javascript">
-			jQuery( function ( $ ) {
-				$( '#posts-filter #post-query-submit' ).parent().append( <?php echo wp_json_encode( $button_html ); ?> );
-			} );
-		</script>
+		<div class="export-card">
+			<div class="export-card__header">
+				<svg width="18" height="24" viewBox="0 0 18 24" fill="none" xmlns="http://www.w3.org/2000/svg">
+					<path d="M11.8387 1.16016H2C1.44772 1.16016 1 1.60787 1 2.16016V21.8053V21.8376C1 22.3899 1.44772 22.8376 2 22.8376H16C16.5523 22.8376 17 22.3899 17 21.8376V5.80532M11.8387 1.16016V5.80532H17M11.8387 1.16016L17 5.80532M4.6129 13.0311V16.1279H9.25806M4.6129 13.0311V9.93435H9.25806M4.6129 13.0311H13.9032M13.9032 13.0311V9.93435H9.25806M13.9032 13.0311V16.1279H9.25806M9.25806 9.93435V16.1279" stroke="#008710" stroke-width="1.5"/>
+				</svg>
+				<div class="export-card__header-title"><?php echo esc_html( __( 'Google Sheets', 'jetpack' ) ); ?></div>
+				<div class="export-card__beta-badge">BETA</div>
+			</div>
+			<div class="export-card__body">
+				<div class="export-card__body-description">
+					<div>
+						<?php echo esc_html( __( 'Export your data into a Google Sheets file.', 'jetpack' ) ); ?>
+						<?php
+						/** Translators: Link to "connect to Google Drive" follows */
+						echo esc_html( __( 'You need to', 'jetpack' ) );
+						echo ' ';
+						// TODO: write/link a support doc for connections
+						echo sprintf(
+							'<a href="%s" title="%s" target="_blank" rel="noopener noreferer">%s</a>',
+							'https://wordpress.com/marketing/connections',
+							esc_attr( __( 'connect to Google Drive', 'jetpack' ) ),
+							esc_html( __( 'connect to Google Drive.', 'jetpack' ) )
```

Comment:
I would recommend making this a single translatable string. It's not going to be possible for translators to translate only "you need to" or "connect to Google Drive" in their language.

Reply:
For anyone interested, we ended up discussing this in p1671122443730359-slack-C01CSBEN0QZ